pkg/neorpc/filters.go

```diff
@@ -104,6 +127,14 @@ func (f *NotificationFilter) Copy() *NotificationFilter {
 	return res
 }
 
+// IsValid implements SubscriptionFilter interface.
+func (f NotificationFilter) IsValid() error {
+	if f.Name != nil && len(*f.Name) > runtime.MaxEventNameLen {
+		return fmt.Errorf("NotificationFilter name parameter must be less than %d. %w", runtime.MaxEventNameLen, ErrInvalidSubscriptionFilter)
```

Comment:
See the https://github.com/nspcc-dev/neo-go/pull/3258#discussion_r1426347845. Firstly `%w`, after then additional error test.